addons/website_slides/models/slide_channel.py

```diff
@@ -277,7 +277,7 @@ def _compute_user_statistics(self):
     @api.depends_context('uid')
     def _compute_can_upload(self):
         for record in self:
-            if record.user_id == self.env.user or self.env.is_superuser():
+            if record.user_id == self.env.user:
```

Comment:
This change seems strange as it is wanted to allow superusers to upload slides (could be discussed, but anyway it was part of the original spec). Why is this line trigerring the readability of slides ? Normally all checks should be done with either real user ACLs, either if sudo (anonymous reading) check that people are member of the channel. I think the fix should probably be done somewhere else...